packages/forms-web-app/src/controllers/selected-appeal/selected-appeal.js

```diff
@@ -0,0 +1,35 @@
+const { VIEW } = require('../../lib/views');
+const { apiClient } = require('../../lib/appeals-api-client');
+const { formatHeadlineData } = require('@pins/common');
+const { determineUser } = require('../../lib/determine-user');
+
+/**
+ * @typedef {import('appeals-service-api').Api.AppealCaseWithAppellant} AppealCaseWithAppellant
+ */
+
+// Shared controller for /appeals/:caseRef and manage-appeals/:caseRef
+exports.get = async (req, res) => {
+	const appealNumber = req.params.appealNumber;
+	const userRouteUrl = req.originalUrl;
+	// TODO update to a new endpoint
+	// this API doesn't really satisfy the spec of AAPD-1247
+	// but I think we should move forward with this to get
+	// this code merged, then create a new endpoint and hook
+	// it up in a later iteration.
+	const caseData = await apiClient.getAppealCaseDataByCaseReference(appealNumber);
```

Comment:
I guess we merge https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/3430 first, and update this?